config/pac-synth.sh

```diff
@@ -0,0 +1,3 @@
+PYTHON_DEPS="maturin"
+MODULE_BUILD_DEPS="rust/1.65.0"
```

Comment:
For future build, use `rust` instead of `rust/1.65.0` unless this requires a very specific rust version? In the latter case, add a comment

Reply:
No specific version needed, will change that.